src/app/components/Image/index.styles.tsx

```diff
@@ -4,10 +4,15 @@ import BASE64_PLACEHOLDER_IMAGE from './base64Placeholder';
 
 const styles = {
   wrapper: css({
-    height: 0,
     overflow: 'hidden',
     position: 'relative',
   }),
+  wrapperFixedAspectRatio: css({
+    height: 0,
+  }),
+  wrapperResponsiveRatio: css({
```

Comment:
```suggestion
  heightFixedAspectRatio: css({
    height: 0,
  }),
  heightResponsiveRatio: css({
```
Just wondering where the `wrapper` naming comes from, might it be better to refer to the css property that is changed?